### PR TITLE
Add hamsternz display port

### LINKED ISSUE #49: Add hamsternz's DisplayPort core example
https://github.com/hamsternz/DisplayPort_Verilog

### PR DESCRIPTION
**Pull request by wtatarski**
Fixes #49

### CODE REVIEW

**Comment by mithro** [comment]
This looks like a pretty great test case. Can you also update the doc at https://docs.google.com/document/d/16L50pyS3RjYStvRKRoWyVac7NoZKyu45fpPQXJqeKYo/edit ?

**Comment by mithro** [comment]
All third party code should be under the `third_party` directory. Can we just use a git submodule here?

**Comment by wtatarski** [comment]
Added hamsternz-displayport as submodule.

**Comment by mithro** [comment]
@acomodi - Just wanted to make sure that you are checking these imports comply with the standards set out in https://github.com/SymbiFlow/ideas/issues/47 !
(PS We probably should have an automated system which checks this....)

**Comment by acomodi** [comment]
@mithro I am actually unsure how to treat the submodules regarding licensing. The hamsternz display port is unmodified and there are no modifications on the source files. For submodules we have no control over their license and the README.fpga-tool-perf should not be needed.

File: src/hamsternz-displayport/constr/vivado-yosys/xc7_a200t_sbg484-1_nexys.xdc [hunk 1]
```diff
@@ -0,0 +1,73 @@
+##Clock Signal
```
Comment (acomodi) [inline src/hamsternz-displayport/constr/vivado-yosys/xc7_a200t_sbg484-1_nexys.xdc:1]: This shouldn't be required anymore

**Review by acomodi — CHANGES_REQUESTED** [review]
Comment in

File: src/hamsternz-displayport/constr/vivado-yosys/xc7_a200t_sbg484-1_nexys.xdc (hunk 1 above)
Comment (kamilrakoczy) [reply src/hamsternz-displayport/constr/vivado-yosys/xc7_a200t_sbg484-1_nexys.xdc]: Removed.

**Review by acomodi — APPROVED** [review]
LGTM

**Pull request merged**